src/main/kotlin/graphics/scenery/Camera.kt

```diff
@@ -227,7 +229,7 @@ open class Camera : DefaultNode("Camera"), HasRenderable, HasMaterial, HasCustom
      *
      * Returns (worldPos, worldDir)
      */
-    fun screenPointToRay(x: Int, y: Int): Pair<Vector3f, Vector3f> {
+    fun screenPointToRay(x: Int, y: Int, camOrigin : Boolean): Pair<Vector3f, Vector3f> {
```

Comment:
@skalarproduktraum Is this switch needed? Line 247 seems to have been a simple mixup or are there cases where we do not want to start the ray directly from the camera?

Reply:
I am confused by this change. The routine is written in the original way in the way it is, because the camera position differs from the point where the ray starts. If you want to go from a screen position (which always resides on the near plane of the camera, which has a finite extent) to a world-space ray, you can't just start from the (zero-dimensional/infinitely small) camera position. Or am I missing something here?

Reply:
When not starting at the zero position of the camera and using the near plane, you can't hit pbjects that are between the near plane and the camera, which happens if ouy move closer to the e.g. plane, that displays a menu in our case. 
Also, for the purpose of scene-detection ray casting, why not use the camera position instead of the near plane?